h2integrate/core/supported_models.py

```diff
@@ -184,10 +186,21 @@
     # Feedstock
     "feedstock_performance": FeedstockPerformanceModel,
     "feedstock_cost": FeedstockCostModel,
+    # Grid
+    "grid_buy_performance": GridBuyPerformanceModel,
+    "grid_sell_performance": GridSellPerformanceModel,
+    "grid_sell_cost": GridSellCostModel,
```

Comment:
This highlights my earlier comment about having one grid. Here the there is only one cost model, but two performance models that may all be needed to model a single tech. I think grid_cost, and grid_performance is all we need.